base_classes/NXinstrument.nxdl.xml

```diff
@@ -21,70 +21,70 @@
 #
 # For further information, see http://www.nexusformat.org
 -->
-<definition xmlns="http://definition.nexusformat.org/nxdl/3.1" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" category="base" type="group" name="NXinstrument" extends="NXobject" xsi:schemaLocation="http://definition.nexusformat.org/nxdl/3.1 ../nxdl.xsd">
-    <doc>
-         Collection of the components of the instrument or beamline.
-         
-         Template of instrument descriptions comprising various beamline components.
-         Each component will also be a NeXus group defined by its distance from the
-         sample. Negative distances represent beamline components that are before the
-         sample while positive distances represent components that are after the sample.
-         This device allows the unique identification of beamline components in a way
-         that is valid for both reactor and pulsed instrumentation.
-    </doc>
-    <field name="name">
-        <doc>
-             Name of instrument
-        </doc>
-        <attribute name="short_name">
-            <doc>
-                 short name for instrument, perhaps the acronym
-            </doc>
-        </attribute>
-    </field>
+<definition xmlns="http://definition.nexusformat.org/nxdl/3.1" category="base"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://definition.nexusformat.org/nxdl/3.1 ../nxdl.xsd"
+	name="NXinstrument" 
+    type="group" extends="NXobject">
+	<doc>
+		Collection of the components of the instrument or beamline.
+		
+		Template of instrument descriptions comprising various beamline components. 
+		Each component will also be a NeXus group defined by its distance from the 
+		sample. Negative distances represent beamline components that are before the 
+		sample while positive distances represent components that are after the sample. 
+		This device allows the unique identification of beamline components in a way 
+		that is valid for both reactor and pulsed instrumentation.
+	</doc>
+	<field name="name">
+		<doc>Name of instrument</doc>
+	    <attribute name="short_name">
+	        <doc>short name for instrument, perhaps the acronym</doc>
+	    </attribute>
+	</field>
     <group type="NXactuator"/>
```

Comment:
This is all heritage from NIAC. I think it's good to already define these in the base class NXinstrument because then the appdefs don't actually have to add them on top.